admin_manual/configuration_server/occ_command.rst

```diff
@@ -1729,13 +1731,13 @@ In the case of a user losing access to the second factor (e.g. lost phone with
 two-factor SMS verification), the admin can try to disable the two-factor
```

Comment:
Maybe we go ahead and add the full command list to the top of the *Two-factor authentication* section while we're at it - e.g.

The ``twofactorauth`` commands are used to configure/manage various aspects of two-factor authentication::
```
      twofactorauth:cleanup Clean up the two-factor user-provider association of an uninstalled/removed provider  
      twofactorauth:enforce Enabled/disable enforced two-factor authentication                                    
      twofactorauth:enable  Enable two-factor authentication for a user                                           
      twofactorauth:disable Disable two-factor authentication for a user                                          
      twofactorauth:state   Get the two-factor authentication (2FA) state of a user.                              
```

(unfortunately GitHub comments don't let one select lines outside the PR AFAIK so I can't make this a suggestion)

Reply:
I would like that. But for the sake of a minimal backport, I would do it separately.